src/utils/initialization.js

```diff
@@ -47,9 +47,9 @@ Now, let's get started!\n`)
     answer = await promptConfirm(`Would you like to create the code and content under your github username, "${org}"?`)
```

Comment:
Would you like to create the code and content repository under your GitHub username?

Reply:
```suggestion
    answer = await promptConfirm(`Would you like to create the code and content repository under your GitHub username, "${org}"?`)
```